app/src/main/kotlin/com/wire/android/ui/sharing/ImportMediaAuthenticatedViewModel.kt

```diff
@@ -368,7 +368,7 @@ class ImportMediaAuthenticatedViewModel @Inject constructor(
         navigationManager.navigate(
             NavigationCommand(
                 NavigationItem.Conversation.getRouteWithArgs(listOf(conversationId)),
-                backStackMode = BackStackMode.REMOVE_CURRENT
+                backStackMode = BackStackMode.CLEAR_TILL_START
```

Comment:
Why are we removing the whole stack? don't we want to keep the previous screens in case the app is already open?

Reply:
I think I remember Kubaz explaining that it was a little bit problematic because we could be at any place in the app before opening this "import media" screen, so we could end up with something like this: home -> conversation -> conversation and that could confuse user as when pressing back he would end up in another conversation instead of home, so either we keep the stack as it was actually opened or we keep the consistency so that the user knows that when pressing back on conversation screen he always ends up on home screen 🤔 Maybe `UPDATE_EXISTED` would be better here, what do you think?

Reply:
I checked how the app behaves in other situations and it looks the same, for instance, if your stack is for instance `Home -> Conversation A -> Conversation A Details` and you click on notification from conversation B, then it becomes `Home ->  Conversation B`. The same if you open the conversation from someone's profile, like: `Home -> Conversation A -> Conversation A Details -> Participants -> Participant X -> Open conversation` then the stack becomes `Home -> Conversation X`.
For me, `CLEAR_TILL_START` makes the navigation behaviour unified throughout the whole app as the user always knows that from "Conversation" screen he/she can always go back to "Home" screen.

Reply:
I guess ultimately this should be a product decision, but until someone else complains about the app navigation not being too intuitive let's go with this approach